### ISSUE TITLE
Extra art frame sizing options

### ISSUE THREAD

**Issue by dalesnail** [Aiue/AQT#44]
_To preface, I understand that this is one option that will take a backseat to other more pressing issues, considering this was an option I randomly asked for ;)_

* **_Hoping we could get "Scale" slider to compliment the current sizing options._** 
    * **Points to consider:**
        * Estimating the pixel range to keep the aspect ratio is tough at times. I've found, that I can't just reduce and increase by a flat amount on both Y and X and reliably keep that ratio. 
        * This is seperate but not by much to the Zoom level, in that it is not tied to the borders of the original image dimensions.
        * Alternatively I think this could be set up in a similar fashion to the Zoom level, with a checkbox that keeps the original ARatio, right beside the X and Y. This could also technically be a solution to the next request. 

* _**So one thing that has thrown me off a bit, and I've had to compensate for is Swapping images for the art option, and resizing it back to it's original dimensions.**_
    * I've been referring back to the original image in windows to resize accordingly, but if it's possible I think that having the image dimensions, and aspect ration snap back to what they are on the original image would be a big QoL improvement. At the moment, the size setting for images stays constant, so swapping between art for previewing is a bit cumbersome at times. 
        * As a note, I'm unsure if this would conflict with the scale option I presented, as keeping the scale setting, but changing back the X and Y seems like it could potentially cause problems. 

Anyway, just wanted to say thanks for the addon, I've been loving it! Glad you got donations up, gonna send over a bit when I get paid this week :)

**Comment by Aiue** [comment]
Scale would be ridiculously easy to implement. Makes me wonder why I didn't right away. Might want to consider it for the main tracker as well.

**Comment by Aiue** [comment]
Might need some extra work. Potentially a checkbox to either use scale or width/height. But that would interfere with the the ability to manually stretch it, so probably something to opt out of.

**Comment by Aiue** [comment]
Not sure whether I managed to include the functionality I wanted with the simple scale slider or not.

**Comment by dalesnail** [comment]
I do like the scaling option, very handy, thank you!

Only thing I might add if possible, is an extension of scale over 100%, maybe to 200%

**Comment by dalesnail** [comment]
I see you added this already, and I can confirm that it's working well!

**Comment by Aiue** [comment]
Yup! Not sure if this issue has anything left for me to add. But yeah, most range sliders will now accept manual input of numbers outside their range.
